tests/test_openai_encoder.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+from pinecone_text.dense.openai_encoder import OpenAIEncoder
+
+DEFAULT_DIMENSION = 1536
+
+
+class TestOpenAIEncoder:
```

Comment:
missing unhappy flows? 

actually we should consider integration tests

Reply:
added a backlog task - https://app.asana.com/0/1204545509946596/1205307556261010/f